dask/blockwise.py

```diff
@@ -113,7 +113,7 @@ class BlockwiseDepDict(BlockwiseDep):
     that the function will be passed a single input object when
     the task is executed (e.g. a single ``tuple`` or ``dict``):
 
-    >>> import pandas as pd  # doctest: +SKIP
+    >>> import pandas as pd
     >>> func = lambda x: pd.read_csv(**x)  # doctest: +SKIP
```

Comment:
Why not take the   `# doctest: +SKIP` off this and the class definition below as well?